operations/host_spawn.go

```diff
@@ -208,11 +210,15 @@ func hostModify() cli.Command {
 				Name:  expireFlagName,
 				Usage: "make host expire like a normal spawn host, in 24 hours",
 			},
+			cli.StringFlag{
+				Name:  addSSHKeyFlag,
+				Usage: "add user defined public key corresponding the `KEY_NAME` to the host's authorized_keys",
```

Comment:
It seems like it would be convenient to also let them upload their public key file directly instead of use the SSH key name in Evergreen. A lot of people don't know their key name, or the key name is the empty string by default (which would count as unspecified input to the CLI).

Reply:
done